solution/ui/regulations/eregs-vite/src/views/Search.vue

```diff
@@ -220,10 +171,32 @@ const setTitle = (query) => {
     document.title = `Search ${querySubString}| Medicaid & CHIP eRegulations`;
 };
 
-// called on load
+getDocsOnLoad = async () => {
+    if (!$route.query.q) {
```

Comment:
Instead of fetching docs after we fetch Subjects (like we do on the Subjects page), here we now fetch docs asynchronously while also fetching Subjects.